app/models/Country.scala

```diff
@@ -29,6 +29,15 @@ object Country {
 
   implicit val format: OFormat[Country] = Json.format[Country]
 
+  def fromCountryCode(countryCode: String): Option[Country] = {
+    euCountries.find(_.code == countryCode)
+  }
+
+  def fromCountryCodeUnsafe(countryCode: String): Country = {
```

Comment:
Unsafe just means can throw an exception versus Eithering and cuts down on getOrElse(throw with a message) clutter.